packages/network/src/streamr-client-protocol.ts

```diff
@@ -0,0 +1,3 @@
+import Protocol from 'streamr-client-protocol'
```

Comment:
Maybe the contents of this file should live in `composition.ts` so instead of
```ts
import { ... } from 'streamr-network/dist/streamr-client-protocol'
```

We can just do 
```ts
import { ... } from 'streamr-network'
```

(Which I find better because the latter is relying on purposefully exported things instead of referring to an arbitrary file within the project.)

We could also consider just adding `streamr-client-protocol` as a dependency of broker and using stuff from there directly, which would be arguably less confusing. Why we haven't done so before? Becuase of concerns of potential version mismatches between network and broker which could lead to weird behaviour.

Reply:
> concerns of potential version mismatches between network and broker which could lead to weird behaviour

Yeah I figured there might be some issues with module-level/class static state or `instanceof` and didn't want to audit every file so just left that part as-is.

Reply:
Though I guess there may be similar issues with client's internal protocol vs broker + network protocol, either versioning or simply a different instance.

Reply:
related https://github.com/streamr-dev/network-monorepo/pull/208#discussion_r714559277